package.json

```diff
@@ -44,54 +44,54 @@
   "homepage": "https://github.com/firebase/firebase-tools",
   "dependencies": {
     "JSONStream": "^1.2.1",
-    "archiver": "^0.16.0",
+    "archiver": "^1.3.0",
     "chalk": "^1.1.0",
-    "cjson": "^0.3.1",
+    "cjson": "^0.5.0",
     "cli-table": "^0.3.1",
     "commander": "^2.8.1",
-    "configstore": "^1.2.0",
-    "cross-spawn": "^4.0.0",
+    "configstore": "^2.1.0",
+    "cross-spawn": "^5.1.0",
     "csv-streamify": "^3.0.4",
     "didyoumean": "^1.2.1",
     "es6-set": "^0.1.4",
     "exit-code": "^1.0.2",
     "filesize": "^3.1.3",
-    "firebase": "2.x.x",
-    "fs-extra": "^0.23.1",
+    "firebase": "3.x.x",
+    "fs-extra": "^1.0.0",
     "fstream-ignore": "^1.0.2",
-    "inquirer": "^0.12.0",
+    "inquirer": "^1.2.3",
     "jsonschema": "^1.0.2",
-    "jsonwebtoken": "^5.4.0",
+    "jsonwebtoken": "^7.3.0",
     "lodash": "^4.6.1",
     "open": "^0.0.5",
-    "ora": "0.2.3",
-    "portfinder": "^0.4.0",
-    "progress": "^1.1.8",
+    "ora": "1.2.0",
```

Comment:
We can't upgrade `ora` because after `0.2.3` dropped support for older versions of node that we support.